src/main.js

```diff
@@ -7,8 +7,11 @@ import axios from "axios";
 import { ValidationProvider, setInteractionMode } from "vee-validate";
 import "./components/validationRules";
 import authService from "./services/authService";
+import subscriptionService from "./services/subscriptionService";
 
+export const bus = new Vue();
```

Comment:
why are you creating new instance of Vue?

Reply:
*EventBus* allows me to emit an event in one component and listen for that event in another.

Reply:
based on our slack conversation, we determined that there is better way to handle it by vuex.